src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -0,0 +1,281 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryMessages;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mercury.LightboxComponentObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+/**
+ * @ownshership: Content West-Wing
+ */
+public class PortableInfoboxObject extends BasePageObject {
+
+  @FindBy(css = "body")
+  private WebElement bodyElement;
+  @FindBy(css = ".portable-infobox")
+  private WebElement infoboxWrapper;
+  @FindBy(css = ".pi-hero .article-image")
+  private WebElement mainImage;
+  @FindBy(css = ".portable-infobox .pi-hero-title")
+  private WebElement title;
+  @FindBy(css = ".portable-infobox .pi-title")
+  private WebElement titleSmallImage;
+  @FindBy(css = ".portable-infobox .pi-expand-button")
+  private WebElement expandButton;
+  @FindBy(css = ".article-content .collapsed")
+  private WebElement infoboxIsCollapsed;
+  @FindBy(css = ".tabber .article-image")
+  private WebElement imageInTabber;
+  @FindBy(css = ".tabber figcaption")
+  private WebElement captionInTabber;
+  @FindBy(css = ".portable-infobox .article-video")
+  private WebElement video;
+  @FindBy(css = ".portable-infobox .article-video figcaption")
+  private WebElement videoCaption;
+  @FindBy(css = ".pi-title img")
+  private WebElement imageInTitle;
+  @FindBy(css = ".portable-infobox .external")
+  private List<WebElement> externalLinks;
+  @FindBy(css = ".pi-item .pi-data-label")
+  private List<WebElement> dataLabels;
+  @FindBy(css = ".pi-item .pi-data-value")
+  private List<WebElement> dataValues;
+  @FindBy(css = ".portable-infobox .reference")
+  private List<WebElement> references;
+  @FindBy(css = ".portable-infobox ul li")
+  private List<WebElement> unorderedLists;
+  @FindBy(css = ".portable-infobox ol li")
+  private List<WebElement> orderedLists;
+  @FindBy(css = ".pi-header")
+  private List<WebElement> headers;
+  @FindBy(css = ".portable-infobox .pi-image img")
+  private List<WebElement> images;
+
+  public PortableInfoboxObject(WebDriver driver) {
+    super(driver);
+  }
+
+  public String getExternalLinkName(int index) {
+    Assertion.assertFalse(externalLinks.isEmpty());
+    wait.forElementVisible(externalLinks.get(index));
+
+    return externalLinks.get(index).getText();
+  }
+
+  public String getUrlFromExternalLinkaAfterPageIsLoaded() {
+    wait.forElementVisible(bodyElement);
+
+    return driver.getCurrentUrl();
+  }
+
+  // TODO: This is not real tap, replace with PerformTouchActions class methods
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)